tests/Test/Dscp/DB/SQLite/Common.hs

```diff
@@ -0,0 +1,107 @@
+{-# LANGUAGE ExplicitForAll             #-}
+{-# LANGUAGE FlexibleContexts           #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase                 #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE PartialTypeSignatures      #-}
+{-# LANGUAGE RankNTypes                 #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE TypeSynonymInstances       #-}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
You can specify it in `package.yaml`.

Reply:
I need no remember how did it get there in the first place. Maybe, I copied it from `Test.Common`.